### ISSUE TITLE
looks like the hosted demo is down.

### ISSUE THREAD

**Issue by Niko-La** [WebReflection/geoshare#1]
Sounds like a cool project. Looks like the site is down right now.  

https://geoshare.now.sh/  

404: NOT_FOUND
Code: DEPLOYMENT_NOT_FOUND

**Comment by WebReflection** [comment]
Yeah, need to update something with now I guess ... not sure this is going to work or if I'll ever do that, but if you're looking for a PWA map able to share location to anyone, check [map](https://github.com/WebReflection/map/#readme) out 👋
